utils/update_build_version.py

```diff
@@ -88,60 +94,69 @@ def deduce_software_version(changes_file):
         for line in f.readlines():
             match = pattern.match(line)
             if match:
-                return match.group(1)
-    raise Exception('No version number found in {}'.format(changes_file))
+                return True, match.group(1)
+    return False, None
```

Comment:
You will need to update the function comment. It just says that a software version number is returned.